index.html

```diff
@@ -77,25 +66,22 @@ <h3>My Favourite Musicians</h3>
             </ol>
 
             <h3>Haiku</h3>
-            The blood moon arises,<br><br>
-            The sky its backdrop darkens,
-            <br><br>
-            Dracula's coffin open.<br><br>
-            <a>
+            <p>
+                The blood moon arises,<br>
+                The sky its backdrop darkens,
+                <br>
+                Dracula's coffin open.<br>
                 <cite>by<span class="author">
                         Slipalong</span>
                 </cite>
-            </a>
+            </p>
         </main>
         <footer>
-            <hr><blockquote>Now, did I ever tell you the tale of the one known as... lucid worm!</blockquote>
-            <cite>
-                <a>
-                    <em>
-                        <span class="author">-Tayah Renaud</span>
-                    </em>
-                </a>
-            </cite>
+            <hr>
+            <blockquote>Now, did I ever tell you the tale of the one known as... lucid worm!<br>
+                <cite>&mdash;<span class="author">Tayah Renaud</span></cite>
+            </blockquote>
```

Comment:
Why the `<a>` here?
Instruction was to use an entity character to make the dash